duckduckgo/privacy-dashboard
# Cosmetic CPM rules

**Reviewer:** @shakyShane 

**CC:** @miasma13 @sammacbeth
**Depends (based) on:** https://github.com/duckduckgo/privacy-dashboard/pull/20

## Description:
Asana task: https://app.asana.com/0/0/1203672644429319/f
This adds support for autoconsent cosmetic rules, and slightly changes the copy.
This branch is based on https://github.com/duckduckgo/privacy-dashboard/pull/20

## Steps to test this PR:

1. To test e2e, you will need to use this in combination with the [latest autoconsent changes](https://github.com/duckduckgo/autoconsent/pull/67), and the corresponding [native changes](https://github.com/more-duckduckgo-org/macos-browser/pull/921)
2.  The CPM screens will look different depending on the type of rule applied:
    - on https://w3cresource.com, it should only "hide" the popup (cosmetic rule)
    - on https://etsy.com, it should show "cookies managed" (non-cosmetic rule)
